StudentManagement/StudentManagement.Console/Program.cs

```diff
@@ -1,18 +1,41 @@
-﻿using StudentManagement.Console;
+﻿using Newtonsoft.Json;
+using StudentManagement.Console;
+using System.Text;
+
+
 
 bool menu = true;
 var students = new List<Student>();
 string name, surname, occupation;
 int age, id;
 
+
+
 do
 {
+
+    string filePath = @"C:\Users\macie\source\repos\StudentManagement\file2.json";
+    var studentJson = JsonConvert.SerializeObject(students);
```

Comment:
you always serialize an empty list of students - what is the point of doing that?